Felienne/hedy
# [UI IMPROVEMENT] Removes copy button from teacher adventure view

**Description**
In this PR we remove the copy button from the ace editor on the teacher adventure view page. As the teacher is the only one able to view the adventure and there is no editor close to copy to the button is redundant. We remove it in this PR.

**Fixes**
This PR fixes #2501

**How to test**
Open one of your teacher adventures and verify that the copy button is no longer visible in the ace editor.
